pkg/plugin/plugin.go

```diff
@@ -404,6 +411,19 @@ func CmdAdd(args *skel.CmdArgs) error {
 	}
 	defer contNetns.Close()
 
+	var origIfName string
+	if netconf.DeviceID != "" {
```

Comment:
Since this block is only relevant for HWOL, it may be confusing for a code-reader who goes through the path of basic OVS flow. Would you please make this clearer by moving the condition to a dedicated function? Maybe something like

```
if hardware offload enabled:
  orig iface name = get vf link name
```

Reply:
Good Idea ! done.